.changeset/wicked-kangaroos-swim.md

```diff
@@ -0,0 +1,5 @@
+---
+"@sveltejs/kit": patch
+---
+
+fix: correctly handle aliases to files in the `.svelte-kit` directory
```

Comment:
actually, maybe it'd be more accurate or comprehensive to say:
```suggestion
fix: correctly handle aliases to files in hidden directories
```

Reply:
I'm not sure that's right. The only issue was that aliases to files in .svelte-kit did not start with a leading ./ so tsconfig considered them to be non-relative paths and complained e.g. "types/routes/src/*"